cpp/include/cudf/io/parquet.hpp

```diff
@@ -582,12 +660,15 @@ class parquet_writer_options_builder {
   /**
    * @brief Sets column chunks file path to be set in the raw output metadata.
    *
-   * @param file_path String which indicates file path.
+   * @param file_paths Vector of Strings which indicates file path. Must be same size as number of
+   * data sinks
    * @return this for chaining.
    */
-  parquet_writer_options_builder& column_chunks_file_path(std::string file_path)
+  parquet_writer_options_builder& column_chunks_file_paths(std::vector<std::string> file_paths)
```

Comment:
This change breaks compilation of benchmarks, which were not updated in this PR: https://github.com/rapidsai/cudf/blob/fc2a32a1576d97a48b0c1c983ef4b31285267e96/cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp#L88

We should ensure that CI compiles the benchmarks even if they don't run.

Reply:
I have the fix in https://github.com/rapidsai/cudf/pull/9776, just needs a small donation of one code review :P